src/lightning_app/core/constants.py

```diff
@@ -65,3 +65,5 @@ def get_lightning_cloud_url() -> str:
 
 DEBUG: bool = lightning_cloud.env.DEBUG
 DEBUG_ENABLED = bool(int(os.getenv("LIGHTNING_DEBUG", "0")))
+
```

Comment:
Let's re-organise the experimental features at the bottom. 

Maybe, we could have a section in the doc about experimental features.